examples/FedRep/fedrep_CIFAR10_resnet18.yml

```diff
@@ -0,0 +1,78 @@
+
+clients:
+    # Type
+    type: simple
+
+    # The total number of clients
+    total_clients: 100
+
+    # The number of clients selected in each round
+    per_round: 40
+
+    # Should the clients compute test accuracy locally?
```

Comment:
According to Section 5.2 of the paper, "We control the heterogeneity of CIFAR10 and CIFAR100 by
assigning different numbers S of classes per client, from among 10 and 100 total classes, respectively." I am not sure if there is a variable to address that. This variable is pretty important because it reflects FedRep's accuracy (Table 1).

Reply:
Good point! In Plato, we generate non-iid distributions in a different way, using the Dirichlet distribution (https://github.com/TL-System/plato/blob/main/plato/samplers/dirichlet.py). With a concentration parameter that is low (say, 0.5), it emulates non-iid distributions in a better and more methodological way than what the paper used. This sampler and its concentration parameter should be specified in the `data` section.